infra-as-code/bicep/orchestration/hubPeeredSpoke/hubPeeredSpoke.bicep

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
Please remove the multi-line comment section at the top of this `.bicep` file.

We have recently moved away from this and have update the contribution guide: https://github.com/Azure/ALZ-Bicep/wiki/Contributing#bicep-file-structure-example

Reply:
Updated in newest commit